### DISCUSSION TITLE
JK inverter BMS - Parallel setup

### DISCUSSION THREAD

**Discussion by at906** [fl4p/batmon-ha#190] [Show and tell]
I recently started migrating my home automation to Home Assistant and I came across this addon and was pleasantly surprised.

My initial attempt wasn't successful, but as I had a new HA installation I went and tried from scratch again. Started with MQTT installation, broker setup, and then went on to BATMON installation. 

I am happy to report everything seem to work fine, with some minor stuff missing ( the inverter BMS has 4 temp sensors vs the old JK active , which had only 2).

I would like to thank the developer for this🙏

I will add a few pictures and can help out anyone who is replicating this.

**Comment by at906** [comment]
![Screenshot_20240124-064219](https://github.com/fl4p/batmon-ha/assets/96946553/8417f450-5f71-4e1d-9244-1ea4094c829c)
![Screenshot_20240124-064252](https://github.com/fl4p/batmon-ha/assets/96946553/5e4c968f-44e4-4339-b2cf-8fbb89d7d245)

**Reply by fl4p** [reply]
you're welcome!

**Comment by CFarwell** [comment]
What issues did you have? I can't keep mine connected via BT.
